sustainml_cpp/examples/CliModules/main.cpp

```diff
@@ -172,21 +194,23 @@ class UserInputPublisher {
         fdds::DomainParticipantFactory::get_instance()->delete_participant(participant);
     }
 
-    void send_sample(int task_id)
+    void send_sample(
+            int task_id)
     {
         UserInputImpl ui;
         GeoLocationImpl geo;
         geo.continent("Continent of task id " + std::to_string(task_id));
         geo.region("Region of task id " + std::to_string(task_id));
         ui.geo_location(geo);
-        ui.problem_description("Problem definition of task id " + std::to_string(task_id));
+        ui.problem_definition("Problem definition of task id " + std::to_string(task_id));
         ui.task_id(task_id);
```

Comment:
Here we may want to fill a task_name including the `to_string(task_id)` within the name, to make it change across tasks